src/style.css

```diff
@@ -1 +1,13 @@
-/* styles go here */
+.fieldset {
+  margin-bottom: 20px;
+  padding-bottom: 0;
+}
+
+.form-field {
```

Comment:
Consider using a more descriptive name for the class. Instead of `.form-field`, you could use `.form-input` or `.form-group` to indicate its purpose more clearly.